Foundation/include/Poco/NumberFormatter.h

```diff
@@ -59,20 +68,36 @@ class Foundation_API NumberFormatter
 		/// right justified and zero-padded in a field
 		/// having at least the specified width.
 
-	static std::string formatHex(int value, bool prefix = false);
+	[[deprecated("use formatHex with options instead")]]
+	static std::string formatHex(int value, bool prefix);
```

Comment:
Can we move all deprecated declarations to end of class definition? This way it is nicer to see this class as first are the ones which we should use.

Reply:
I intentionally left them where they were before for the users to easily see the new functions that they should use instead. I can move them to the end of the class, no problem.

Reply:
Resolved in new commit.